backend/editor/api.py

```diff
@@ -1,11 +1,23 @@
+"""
+Taxonomy Editor Backend API
+"""
+# Required imports
+#------------------------------------------------------------------------#
 from datetime import datetime
 
 # FastAPI
-from fastapi import FastAPI, status, Response, Depends, Header, HTTPException, Cookie
-from fastapi.responses import JSONResponse
-from fastapi.security import OAuth2PasswordBearer, OAuth2PasswordRequestForm
+from fastapi import FastAPI, status, Response, Request, HTTPException
 from fastapi.middleware.cors import CORSMiddleware
 
+# Data model imports
+from models import Header, Footer
+
+# DB helper imports
+from entries import initialize_db, shutdown_db
+from entries import get_all_nodes, get_nodes
+from entries import update_nodes

```

Comment:
sorry I didn't spot it before but:

* you need a __init__.py file in editor folder (this will make the editor folder a package)
* use "relative" import here:

```suggestion
from .models import Header, Footer

# DB helper imports
from .entries import initialize_db, shutdown_db
from .entries import get_all_nodes, get_nodes
from .entries import update_nodes
```

It's important to avoid collision with eventual system level packages.

Reply:
A init.py file is already present in the repo. For testing, it was easier to not have relative imports.